keadm/cmd/keadm/app/cmd/reset.go

```diff
@@ -124,23 +124,11 @@ func RemoveContainers(isEdgeNode bool, execer utilsexec.Interface) error {
 		return nil
 	}
 
-	criSocketPath, err := utilruntime.DetectCRISocket()
-	if err != nil {
-		return err
-	}
-	fmt.Println("[Test] Got criSocketPath", criSocketPath)
+	criSocketPath, _ := utilruntime.DetectCRISocket()
 
-	containerRuntime, err := utilruntime.NewContainerRuntime(execer, criSocketPath)
-	if err != nil {
-		return err
-	}
-	fmt.Println("[Test] Got containerRuntime", containerRuntime)
+	containerRuntime, _ := utilruntime.NewContainerRuntime(execer, criSocketPath)
 
-	containers, err := containerRuntime.ListKubeContainers()
-	if err != nil {
-		return err
-	}
-	fmt.Println("[Test] Got containers", containers)
+	containers, _ := containerRuntime.ListKubeContainers()
```

Comment:
handle error and use `klog` to print out.

Reply:
Added.